File: 01_Functions_and_Execution_Context/05-getRemainder.js
// Create a function getRemainder that takes two numbers and returns the remainder (value left over from division) of dividing the larger number by the smaller one.

// Hint- You have no gaurantee which number will be the bigger number, how can you determine which is which?

function getRemainder(num1, num2) {
  //your code goes here...
	return num2 > num1 ? num2 % num1 : num1 % num2;
}


// console.log(getRemainder(17, 5)); //=> 2
// console.log(getRemainder(20, 5)); //=> 0
// console.log(getRemainder(8, 22)); //=> 6
// console.log(getRemainder(7, 42)); //=> 0

File: 01_Functions_and_Execution_Context/05-getRemainder.py
# Create a function getRemainder that takes two numbers and returns the remainder (value left over from division) of dividing the larger number by the smaller one.

# Hint- You have no guarantee which number will be the bigger number, how can you determine which is which?


def getRemainder(num1, num2):
  if num2 > num1:
    return num2 % num1
  else:
    return num1 % num2


print(getRemainder(17, 5)) #=> 2
print(getRemainder(20, 5)) #=> 0
print(getRemainder(8, 22)) #=> 6
print(getRemainder(7, 42)) #=> 0